testing/custom/run_gpu_operator_with_dtk.sh

```diff
@@ -0,0 +1,69 @@
+#! /usr/bin/env bash
+
+set -o pipefail
+set -o errexit
+set -o nounset
+
+set -x
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd ${THIS_DIR}/../..
+
+if [ -z "${ARTIFACT_DIR:-}" ]; then
+    export ARTIFACT_DIR="/tmp/ci-artifacts_$(date +%Y%m%d)"
+    echo "Using '$ARTIFACT_DIR' to store the test artifacts (default value for ARTIFACT_DIR)."
+else
+    echo "Using '$ARTIFACT_DIR' to store the test artifacts."
+fi
+
+# ensure that the cluster is not entitled
+echo "INFO: Testing if the cluster is already entitled ..."
+if ./run_toolbox.py entitlement test_cluster --no_inspect; then
+    echo "INFO: Cluster already entitled :("
+    exit 1
+fi
+echo "###"
+echo "### Cluster is not entitled"
+echo "###"
+
+# install the NFD Operator
+./run_toolbox.py nfd_operator deploy_from_operatorhub
+
+# add a GPU node to the cluster
+./run_toolbox.py cluster set_scale g4dn.xlarge 1
+
+prepare_dtk_imagestream() {
```

Comment:
Why is this function needed? Isn't DTK already part of OCP?

Reply:
the image is there, the `imagestream` isn't